_data/simple-icons.json

```
@@ -15138,7 +15138,7 @@
         },
         {
             "title": "Stripe",
-            "hex": "008CDD",
+            "hex": "7069FE",
```

```suggestion
            "hex": "635BFF",
```

`#635BFF` is the hex code I'm seeing used in the logo assets. Where did you get `#7069FE` from?